### ISSUE TITLE
'kpod push' doesn't use default transport

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [projectatomic/libpod#40]
If you don't specify a transport, kpod push doesn't default to a transport.  It probably should default to "docker://" if a transport is not provided.  A similar fix was made to Buildah at: https://github.com/projectatomic/buildah/pull/260

In the examples below, "docker://" is not specified as the transport in the first call and kpod push fails.  It then completes when specified.

```
# kpod push --cert-dir /root/auth docker.io/library/alpine:latest dockreg.redhat.com:5000/my-alpine2
error getting destination imageReference for "dockreg.redhat.com:5000/my-alpine2": Invalid image name "dockreg.redhat.com:5000/my-alpine2", unknown transport "dockreg.redhat.com"
[root@localhost tmp]# kpod push --cert-dir /root/auth docker.io/library/alpine:latest docker://dockreg.redhat.com:5000/my-alpine2
Getting image source signatures

```

**Comment by rhatdan** [comment]
Assigned.

**Comment by TomSweeneyRedHat** [comment]
Fixed by #42
